### DISCUSSION TITLE
DEM and DSM files

### DISCUSSION THREAD

**Discussion by paula-lelis** [UMEP-dev/UMEP#414] [General]
Hello, I have DEM and DSM files for a small zone with information only inside the perimeter's zone. To run UMEP properly, I understand that is necessary to simulate a square grid. In this case, I can insert my data into a big square that fits my perimeter. I would like to know if UMEP accept those areas with no data and if this affects the simulation. In case it is possible, I would like to know how to set this no data in the rasters. Best regards, Paula.

ps: In the attached file is possible to see the DSM available file, the square and the perimeter zone
![area](https://user-images.githubusercontent.com/102325941/167140831-34b03c9f-62d3-4b38-8a2b-d847de1212b1.JPG)

**Comment by biglimp** [comment]
The model domain does not have to be square. It can be rectangular also. Set the no data to zero and if you are using ground cover, set the area to e.g. water or paved, not Nodata.

**Reply by paula-lelis** [reply]
Hello biglimp, thank you very much!
